scripts/setupvars/setupvars.bat

```diff
@@ -63,7 +63,7 @@ set "PATH=%OPENVINO_LIB_PATHS%;%PATH%"
 :: Check if Python is installed
 set PYTHON_VERSION_MAJOR=3
 set MIN_REQUIRED_PYTHON_VERSION_MINOR=6
-set MAX_SUPPORTED_PYTHON_VERSION_MINOR=9
+set MAX_SUPPORTED_PYTHON_VERSION_MINOR=10
```

Comment:
@p-wysocki do you leave it here on purpose? Should **setupvars** be affected only after full CI migration?

Reply:
Same change has been done in https://github.com/openvinotoolkit/openvino/pull/12578, however I see no harm if setupvars bump gets merged earlier. The official Python 3.10 support comes with aforementioned PR.